modular_zubbers/code/modules/storyteller/gamemode.dm

```diff
@@ -231,7 +231,12 @@ SUBSYSTEM_DEF(gamemode)
 		else if (observers && isobserver(player))
 			candidate_candidates += player
 		else if (living_players && isliving(player))
-			candidate_candidates += player
+			if(!allow_offstation)
+				if(!is_station_level(player.z)) // Should stop all the silly incorrect rolls. Also stops someone in piss nowhere space from rolling (Wasted roll if you ask me)
```

Comment:
mediocre way of doing it

try using GLOB.manifest.locked to iterate through people to potentially give antag to, its a strict "who is crew and exists at the moment" list

Reply:
And you would be correct